### ISSUE TITLE
you don't need postinstall module

### ISSUE THREAD

**Issue by kapouer** [ethnamed/web3t#2]
Hi
postinstall module is meant to be called from postinstall script, and configured with some fields in package.json.
Since there is absolutely nothing in your module doing that, i guess the dependency on postinstall is a mistake.

**Comment by askucher** [comment]
Yeah! I just added it because trying to make the quick solution for the `bitcore-message` problem when it loads few instances of `bitcore` libraries. Honestly, I do not know how to fix this (https://github.com/bitpay/bitcore-lib/issues/21) problem quickly. 

But hope will remove this soon

**Comment by kapouer** [comment]
Seriously, you don't need, in package.json, the line `"postinstall": "0.4.2"` *at all*.
The "scripts" "postinstall" is gonna run without it just fine.
